src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -1102,6 +1160,13 @@ bool isLongLivedCaptureReferenceSyntax(SyntaxNode captureReferenceSyntax)
                                         return true;
                                     }
                                     break;
+
+                                case CSharp.SyntaxKind.ConditionalAccessExpression:
+                                    if (((CSharp.Syntax.ConditionalAccessExpressionSyntax)syntax.Parent).Expression == syntax)
```

Comment:
>if (((CSharp.Syntax.ConditionalAccessExpressionSyntax)syntax.Parent).Expression == syntax) [](start = 36, length = 90)

I think we should see if we want to fix the builder instead. Looking at the added ```NestedConditionalAccess``` unit-test, the life-time of capture 0 is extended unnecessarily.

Reply:
In any case I do not think we should be treating it as long lived (i.e. something that is known to span multiple statements). So the relaxation should probably be applied elsewhere

---
In reply to: [529818891](https://github.com/dotnet/roslyn/pull/49534#discussion_r529818891) [](ancestors = 529818891)

Reply:
After much offline investigation and discussion, I will be tightening the assertion, but we will be keeping the builder as is. It's not ideal behavior, but would require significant effort to make ideal with significant maintenance burden, and is not worth it for this case.

---
In reply to: [530039773](https://github.com/dotnet/roslyn/pull/49534#discussion_r530039773) [](ancestors = 530039773,529818891)